modules/google-maps/src/google-maps-overlay.js

```diff
@@ -186,9 +195,18 @@ export default class GoogleMapsOverlay {
       });
 
       withParameters(gl, GL_STATE, () => {
-        deck._drawLayers('google-vector', {
-          clearCanvas: false
-        });
+        // Manually draw layers to avoid clearing basemap
+        deck._drawLayers('google-vector', {clearCanvas: false});
+
+        // Camera changed, will trigger a map repaint right after this. Clear any change flag
+        // triggered by setting viewState so that deck does not request a second repaint
+        deck.needsRedraw({clearRedrawFlags: true});
+
+        // Invoke animation loop callback that was
+        // intercepted above. We need to do this here to
+        // make sure any GL state changes are contained to
+        // the _onDrawVector lifecycle method.
+        deck.animationLoop.onRender();
```

Comment:
This will check `needsRedraw` and if so call `requestRedraw` again inside `_customRender`. You end up with a double rerender with every layer update.